src/main/java/org/craftercms/engine/util/spring/servlet/AppContextAwareServletWrappingController.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.craftercms.engine.util.spring.servlet;
+
+import java.util.Enumeration;
+import java.util.Properties;
+import javax.servlet.Servlet;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletContext;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.BeanNameAware;
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.AbstractController;
+
+
+/**
+ * Based on {@link org.springframework.web.servlet.mvc.ServletWrappingController}
+ *
+ * Note: the class can't be extended because all fields are private
+ *
+ * @author joseross
+ * @since 3.1.6
+ */
+public class AppContextAwareServletWrappingController extends AbstractController
```

Comment:
Is this class used? I don't see it referenced anywhere else in the PR and the XML config seems to be using: `org.springframework.web.servlet.mvc.ServletWrappingController`

Reply:
It's not, I will remove it